Composer/packages/client/src/utils/navigation.ts

```diff
@@ -112,3 +114,18 @@ export const openInEmulator = (url, authSettings: { MicrosoftAppId: string; Micr
   }&msaAppPassword=${encodeURIComponent(authSettings.MicrosoftAppPassword)}`;
   document.body.appendChild(i);
 };
+
+const modeMap: { [page in PageMode]?: string } = {
```

Comment:
Could the modeMap[page] be a selector?

Reply:
Since I no longer have the page mode in the Recoil store, there's a better way to do this that doesn't need this modeMap at all.